src/main/arm/mainTemplate.json

```diff
@@ -668,6 +668,7 @@
             }
         },
         {
+            "condition": "[parameters('configureIHS')]",
```

Comment:
The change is non-related to the PR topic, it's just tiny enough to be included in be shipped with the PR.